### PR TITLE
chore(KIC): update references for v2.12.x

### PR DESCRIPTION
**Pull request by tao12345666333**
### Description

Auto-update the references for KIC v2.12.x

 

### Testing instructions

Preview link: 

* https://deploy-preview-6685--kongdocs.netlify.app/kubernetes-ingress-controller/2.12.x/references/custom-resources/
* https://deploy-preview-6685--kongdocs.netlify.app/kubernetes-ingress-controller/2.12.x/references/cli-arguments/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_src/kic-v2/references/cli-arguments-2.12.x.md [hunk 1]
```diff
@@ -74,9 +77,9 @@ and not CLI flags.
 | `--kong-admin-tls-client-key-file` | `string` | MTLS client key file for authentication. |  |
 | `--kong-admin-tls-server-name` | `string` | SNI name to use to verify the certificate presented by Kong in TLS. |  |
 | `--kong-admin-tls-skip-verify` | `bool` | Disable verification of TLS certificate of Kong's Admin endpoint. | `false` |
-| `--kong-admin-token` | `string` | The {{site.ee_product_name}} RBAC token used by the controller. |  |
+| `--kong-admin-token` | `string` | The Kong Enterprise RBAC token used by the controller. |  |
```
Comment (cloudjumpercat) [inline app/_src/kic-v2/references/cli-arguments-2.12.x.md:80]:
```suggestion
| `--kong-admin-token` | `string` | The {{site.ee_product_name}} RBAC token used by the controller. |  |
```
This variable is correct in this instance (the name changed a little while ago to "Kong Gateway Enterprise", which is what this variable renders as)

File: app/_src/kic-v2/references/cli-arguments-2.12.x.md [hunk 2]
```diff
@@ -74,9 +77,9 @@ and not CLI flags.
...
 | `--kong-admin-url` | `stringSlice` | Kong Admin URL(s) to connect to in the format "protocol://address:port". More than 1 URL can be provided, in such case the flag should be used multiple times or a corresponding env variable should use comma delimited addresses. | `[http://localhost:8001]` |
-| `--kong-workspace` | `string` | {{site.ee_product_name}} workspace to configure. Leave this empty if not using Kong workspaces. |  |
+| `--kong-workspace` | `string` | Kong Enterprise workspace to configure. Leave this empty if not using Kong workspaces. |  |
```
Comment (cloudjumpercat) [inline app/_src/kic-v2/references/cli-arguments-2.12.x.md:82]:
```suggestion
| `--kong-workspace` | `string` | {{site.ee_product_name}} workspace to configure. Leave this empty if not using Kong workspaces. |  |
```

File: app/_src/kic-v2/references/custom-resources-2.12.x.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ KongClusterPlugin is the Schema for the kongclusterplugins API.
...
 | `protocols` _[KongProtocol](#kongprotocol) array_ | Protocols configures plugin to run on requests received on specific protocols. |
-| `ordering` _PluginOrdering_ | Ordering overrides the normal plugin execution order. It's only available on {{site.ee_product_name}}. `<phase>` is a request processing phase (for example, `access` or `body_filter`) and `<plugin>` is the name of the plugin that will run before or after the KongPlugin. For example, a KongPlugin with `plugin: rate-limiting` and `before.access: ["key-auth"]` will create a rate limiting plugin that limits requests _before_ they are authenticated. |
+| `ordering` _PluginOrdering_ | Ordering overrides the normal plugin execution order. It's only available on Kong Enterprise. `<phase>` is a request processing phase (for example, `access` or `body_filter`) and `<plugin>` is the name of the plugin that will run before or after the KongPlugin. For example, a KongPlugin with `plugin: rate-limiting` and `before.access: ["key-auth"]` will create a rate limiting plugin that limits requests _before_ they are authenticated. |
```
Comment (cloudjumpercat) [inline app/_src/kic-v2/references/custom-resources-2.12.x.md:43]:
```suggestion
| `ordering` _PluginOrdering_ | Ordering overrides the normal plugin execution order. It's only available on {{site.ee_product_name}}. `<phase>` is a request processing phase (for example, `access` or `body_filter`) and `<plugin>` is the name of the plugin that will run before or after the KongPlugin. For example, a KongPlugin with `plugin: rate-limiting` and `before.access: ["key-auth"]` will create a rate limiting plugin that limits requests _before_ they are authenticated. |
```

File: app/_src/kic-v2/references/custom-resources-2.12.x.md [hunk 4]
```diff
@@ -107,7 +107,7 @@ KongPlugin is the Schema for the kongplugins API.
...
 | `protocols` _[KongProtocol](#kongprotocol) array_ | Protocols configures plugin to run on requests received on specific protocols. |
-| `ordering` _[PluginOrdering](#pluginordering)_ | Ordering overrides the normal plugin execution order. It's only available on {{site.ee_product_name}}. `<phase>` is a request processing phase (for example, `access` or `body_filter`) and `<plugin>` is the name of the plugin that will run before or after the KongPlugin. For example, a KongPlugin with `plugin: rate-limiting` and `before.access: ["key-auth"]` will create a rate limiting plugin that limits requests _before_ they are authenticated. |
+| `ordering` _[PluginOrdering](#pluginordering)_ | Ordering overrides the normal plugin execution order. It's only available on Kong Enterprise. `<phase>` is a request processing phase (for example, `access` or `body_filter`) and `<plugin>` is the name of the plugin that will run before or after the KongPlugin. For example, a KongPlugin with `plugin: rate-limiting` and `before.access: ["key-auth"]` will create a rate limiting plugin that limits requests _before_ they are authenticated. |
```
Comment (cloudjumpercat) [inline app/_src/kic-v2/references/custom-resources-2.12.x.md:110]:
```suggestion
| `ordering` _[PluginOrdering](#pluginordering)_ | Ordering overrides the normal plugin execution order. It's only available on {{site.ee_product_name}}. `<phase>` is a request processing phase (for example, `access` or `body_filter`) and `<plugin>` is the name of the plugin that will run before or after the KongPlugin. For example, a KongPlugin with `plugin: rate-limiting` and `before.access: ["key-auth"]` will create a rate limiting plugin that limits requests _before_ they are authenticated. |
```

**Review by cloudjumpercat — APPROVED** [review]
LGTM! Thanks for making these updates!

File: app/_src/kic-v2/references/cli-arguments-2.12.x.md (hunk 1 above)
Comment (tao12345666333) [reply app/_src/kic-v2/references/cli-arguments-2.12.x.md]: Thanks! Should I update the origin template in KIC's repo? (This file was generated from a template)

**Pull request merged**